index.js

```diff
@@ -76,7 +76,7 @@ Limiter.prototype.get = function (fn) {
         if (err) return fn(err);
 			  // If the request has failed, it means the values already
 			  // exist in which case we need to get the latest values.
-        if (!res || !res[0]) return mget();
+        if (!res || !res[0] || !res[0][1]) return mget();
```

Comment:
The only problem that I see is that you rely on the fact that node_redis returns a string result.

I think that a better way would be to check if res[0] is an array, if so, check if res[0][1] exists.
what do you think?